scripts/generate-docs-start-pipeline.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+# Implements https://circleci.com/docs/api/v2/#trigger-a-new-pipeline
+
+import argparse
+import os
+import requests
+import sys
+
+
+def TriggerPipeline(slug, token, branch, params):
+    url = "https://circleci.com/api/v2/project/github/%s/pipeline" % (slug)
+
+    headers = {
+        "Content-Type": "application/json",
+        "Accept": "application/json",
+    }
+
+    data = {
+        "parameters": params
+    }
+
+    if branch:
+        data["branch"] = branch
+
+    r = requests.post(url, auth=(token, ""), headers=headers, json=data)
+
+    if r.status_code != 201 and r.status_code != 200:
+        print("Error triggering the CircleCI: %s." % r.json()["message"])
+        sys.exit(1)
+
+
+def Main():
```

Comment:
```suggestion
def main():
```

https://www.datacamp.com/community/tutorials/pep8-tutorial-python-code